traffic_ops/testing/api/v5/federation_resolvers_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-util/assert"
 	"github.com/apache/trafficcontrol/traffic_ops/testing/api/utils"
 	"github.com/apache/trafficcontrol/traffic_ops/toclientlib"
+
```

Comment:
nit: I don't think we need a blank line here, because it belongs to the same import block